src/commands/download-p5.cjs

```diff
@@ -0,0 +1,55 @@
+/// @ts-check
+const https = require('https');
+const fs  = require('fs')
+const upath = require('upath');
+
+
+const command = "download-p5 [dir]";
+const desc = "Clone or update p5.js git repository";
+
+// exports.builder = {
+//   dir: {
+//     type: 'string',
+//     default: joinSafe(__dirname, '../../p5'),
+//     describe: 'Destination directory for p5.js'
+//   },
+//   repo: {
+//     type: 'string',
+//     default: 'https://github.com/processing/p5.js.git',
+//     describe: 'URL to p5.js repository (use https)'
+//   },
+//   branch: {
+//     type: 'string',
+//     default: 'main',
+//     describe: 'branch to check out and update from'
+//   },
+//   silent: {
+//     type: 'boolean',
+//     describe: 'Suppress status messages',
+//     default: false
+//   }
+// };
+
+// exports.handler = async args => {
+//   const log = !args.silent;
+
+// https://stackoverflow.com/a/13544465
+fs.mkdirSync(upath.joinSafe(__dirname, "../../p5/docs/reference/"), {
```

Comment:
This path is unnecessarily complex now that git is not used for downloading this data. It would be best to decide a new path and change generate to use this new path.

Reply:
Any suggestions for the new path?